scripts/source-maintenance.sh

```diff
@@ -721,13 +723,25 @@ collectAuthors ()
         return 0 # successfully did nothing, as requested
     fi
 
-    vettedCommitPhraseRegex='[Rr]eference point for automated CONTRIBUTORS updates'
+    vettedPhraseRegex='[Rr]eference point for automated CONTRIBUTORS updates'
+
+    vettedByPr=false
+    if test -n "${PULL_REQUEST_NUMBER}"
+    then
+        vettedByPr=`gh pr view $PULL_REQUEST_NUMBER --json body --jq ".body | test(\"${vettedPhraseRegex}\")"`
```

Comment:
Using single quotes in test does not work:
```
... --jq ".body | test('${vettedCommitPhraseRegex}')"

failed to parse jq expression (line 1, column 14)
    .body | test('point')
                 ^  unexpected token "'"

```
The proper [way](http://stackoverflow.com/questions/40027395/passing-bash-variable-to-jq) to pass a shell variable to jq is via --arg argument. But it is not possible to do this here because gh does not have an additional --arg for its --jq parameter (but they suggest to [add](https://github.com/cli/cli/issues/10263) this feature). So we can either leave it as is, or, as it was suggested in the first reference, export vettedPhraseRegex and use it via `test(env.vettedPhraseRegex)`.

Reply:
Thank you for investigating this. I did not know that jq does not support single quotes for string constants. Let's leave this code as is despite its escaping complexities, at least for now.